### ISSUE TITLE
Forge server update caused Tps lag with jas??

### ISSUE THREAD

**Issue by skittishtrigger** [ProjectZulu/JustAnotherSpawner#140]
I recently updated my forge to 1448 (this is the only thing that has changed recently) before this there was 0 tps lag. My average tps over a 48h period was 19.48 with spikes down as low as 17.  I ran a /sampler start/stop/export (Don't remember how long but it was at min an hour cause I forgot I had it running) and tracked the lag down to http://i.imgur.com/yhc9gQR.jpg .  To download the full nps file I hosted it for you at solder.storagedragon.info/modpack/mods/lagcheck.nps  .  Hopefully it is something I have overlooked or missed. would appreciate your advice or any info you can provide so I can squelch this. 

tl:dr
**************************************************************************************************
issue: http://i.imgur.com/yhc9gQR.jpg
full log: http://solder.storagedragon.info/modpack/mods/lagcheck.nps
mods list: http://www.technicpack.net/modpack/technainteasy.650138/mods
Forge: 10.13.4.1448
FML: v7.10.99.99
MCP: v9.05
Server: Ubuntu 14.10 headless (dedicated)
Ram: 10g

I hope that is enough/correct info.
As an addendum, I should probably check what forge/java version this is built against.

**Comment by skittishtrigger** [comment]
If this has to do with forge version and not the mod itself feel free to close this issue. I have opened issue on https://github.com/skittishtrigger/TechnAintEasy/issues/2 modpack issue reporting if this is the case.

**Comment by Crudedragos** [comment]
I haven't had time lately to look into this, but wanted to say that I have seen it.

I can't imagine anything in a forge update causing an issue.

What are the player and entity counts you have for your test conditions?

**Comment by Crudedragos** [comment]
Issue is likely here https://github.com/ProjectZulu/JustAnotherSpawner/blob/master/src/main/java/jas/spawner/modern/EntityDespawner.java#L25

CountInfo is being calculated for every entity that updates, I cannot see I reason why that is required.

**Comment by Crudedragos** [comment]
Reopen if issue persists
